cg/cli/demultiplex/add.py

```diff
@@ -47,6 +47,7 @@ def add_flow_cell_cmd(context: CGConfig, flow_cell_name: str, bcl_converter: str
     demux_results: DemuxResults = DemuxResults(
         demux_dir=demux_results_path, flow_cell=flow_cell, bcl_converter=bcl_converter
     )
+    ## check for machine and use separate
```

Comment:
separate what?

Reply:
this was an artefact from a previous conversation with Sebastion - removed it!